pkg/controller/controller.go

```diff
@@ -163,7 +163,14 @@ func sendEvent(obj, oldObj interface{}, c *config.Config, notifiers []notify.Not
 
 	// check for siginificant Update Events in objects
 	if eventType == config.UpdateEvent {
-		updateMsg := utils.Diff(oldObj, obj)
+		updatesettingAll, existsAll := utils.AllowedUpdateEventsMap[utils.KindNS{Resource: kind, Namespace: "all"}]
+		updatesettingNS, existsNS := utils.AllowedUpdateEventsMap[utils.KindNS{Resource: kind, Namespace: objectMeta.Namespace}]
+		var updateMsg string
+		if existsAll {
+			updateMsg = utils.Diff(oldObj, obj, updatesettingAll)
+		} else if existsNS {
+			updateMsg = utils.Diff(oldObj, obj, updatesettingNS)
+		}
```

Comment:
We can avoid code duplication here.
```
// Check if all namespaces allowed
updateSetting, exist := utils.AllowedUpdateEventsMap[utils.KindNS{Resource: kind, Namespace: "all"}]
fi !exist {
    // Check if specified namespace is allowed
    updateSetting, exist = utils.AllowedUpdateEventsMap[utils.KindNS{Resource: kind, Namespace: objectMeta.Namespace}]
}
if exist {
    updateMsg = utils.Diff(oldObj, obj, updatesettingNS
}
```

Reply:
Hi @PrasadG193, Made the required changes. Please review.